Resources/config/services.yml

```diff
@@ -1 +1,16 @@
-#services:
+services:
```

Comment:
There is no such files in bundle : what is the purpose?

Reply:
Services definition is necessary for use API classes.
Symfony doesn't ban the services in bundles: 
https://symfony.com/doc/current/bundles/extension.html
https://symfony.com/doc/current/bundles/best_practices.html#directory-structure